easybuild/easyconfigs/k/KneadData/KneadData-0.6.1-foss-2017a-Python-2.7.13.eb

```diff
@@ -0,0 +1,35 @@
+easyblock = 'PythonPackage'
+
+name = 'KneadData'
+version = '0.6.1'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://pypi.python.org/pypi/kneaddata'
+description = """KneadData is a tool designed to perform quality control on metagenomic
+ sequencing data, especially data from microbiome experiments."""
+
+toolchain = {'name': 'foss', 'version': '2017a'}
+
+source_urls = [PYPI_LOWER_SOURCE]
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['46540a22bebc84919a71138c419c69f68e9f5e35404587a2e31a3a3e53c66077']
+
+dependencies = [
+    ('Python', '2.7.13'),
+    ('Trimmomatic', '0.36'),
```

Comment:
@tutufan You'll need to specify the `'-Java-1.8.0_162'` `versionsuffix` here as 3rd element in the tuple. Same for `FastQC`:

```python
('Trimmomatic', '0.36', '-Java-1.8.0_162'),
...
('FastQC', '0.11.5', '-Java-1.8.0_162'),
```